cypress/plugins/nodeManager/plugin.ts

```diff
@@ -108,25 +111,34 @@ class Node {
     this.httpPort = options.id;
     this.peerPort = options.id;
     this.proxyBinaryPath = path.join(ROOT_PATH, options.proxyBinaryPath);
+    this.storagePath = path.join(CYPRESS_WORKSPACE_PATH, uuid.v4());
   }
 
   async start() {
     this.logger.log("starting node");
 
+    fs.mkdirsSync(this.storagePath);
+
+    const envCopy: NodeJS.ProcessEnv = {};
+    Object.assign(envCopy, global.process.env);
+    envCopy["RAD_HOME"] = this.storagePath;
+
     const process = childProcess.spawn(
       this.proxyBinaryPath,
       [
         "--test",
+        "--test-use-rad-home",
         "--http-listen",
         `${HOST}:${this.httpPort}`,
         "--peer-listen",
         `${HOST}:${this.peerPort}`,
       ],
-      {}
+      { env: envCopy }
```

Comment:
So how come both the env and the option are needed?

Reply:
Because it's still WIP. Will clean it up once I have everything working.

Reply:
For more details see: https://github.com/radicle-dev/radicle-upstream/pull/1589/files#r574634262.